### DISCUSSION TITLE
Install Lux Widget

### DISCUSSION THREAD

**Discussion by ampledata** [microsoft/PlanetaryComputer#54] [General]
Hi PC Team,

If possible, can you add, or is there a way for me to add, the Lux widget? 

https://blog.dominodatalab.com/faster-data-exploration-in-jupyter-through-the-lux-widget

**Comment by TomAugspurger** [comment]
I haven't used Lux before. In theory, you're able to add jupyterlab plugins at runtime by installing them and then reloading the page, if you want to try that.
